libs/challenge-registry/challenge-search/src/lib/challenge-search-filters-values.ts

```diff
@@ -138,3 +138,21 @@ export const challengeIncentiveTypesFilterValues: FilterValue[] = [
 export const challengePlatformFilterValues: FilterValue[] = [];
 
 export const challengeOrganizationFilterValues: FilterValue[] = [];
+
+export const challengeSortFilterValues: FilterValue[] = [
+  {
+    value: 'startDate',
+    label: 'Starting Soon',
+    active: false,
+  },
+  {
+    value: 'endDate',
+    label: 'Closing Soon',
+    active: false,
+  },
+  {
+    value: 'starredCount',
+    label: 'Popularity',
```

Comment:
Rather than "Popularity", can we use "Stars" or "Most favorited" instead?

"Popularity" might be a bit vague between number of stars vs. number of participants vs. number of submissions, etc. (or maybe I'm just overthinking this 🙃 )

Reply:
I agree, "Most favorited" or "Most favorites" sounds good to me. @tschaffter What do you think?

Reply:
A more descriptive name sounds good. What about "Most starred" (shorter and easier to pronounce).

Reply:
cool, let's do "Most Starred" for now. Thanks for suggestions.